supplementary/devices/taoshan.rst

```diff
@@ -0,0 +1,67 @@
+
+Taoshan - Sony Xperia L
+===========================
+
+The device is quite weak to run Halium. But it should be possible.
+
+Status
+------
+
+Halium
+^^^^^^
+A port for Halium 7.1 is being worked on by `Jonatan Hatakeyama Zeidler <https://github.com/jonnius>`_ (`Device Tree <https://github.com/jonnius/android_device_sony_taoshan>` and `Kernel <https://github.com/jonnius/android_kernel_sony_msm8930/tree/halium>`). Halium 7.1 builds, but does not install using halium-install due to twrp busybox issues. Using halium-install from JBB the halium reference rootfs can be installed, but it does not seem to start.
```

Comment:
Please, capitalize words that were meant to be capitalized for the sake of sanity. I.e. TWRP, BusyBox, etc